pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -76,172 +97,306 @@ type JaegerStatus struct {
 // Jaeger is the Schema for the jaegers API
 // +k8s:openapi-gen=true
 type Jaeger struct {
-	metav1.TypeMeta   `json:",inline"`
+	metav1.TypeMeta `json:",inline"`
+
+	// +optional
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   JaegerSpec   `json:"spec,omitempty"`
+	// +optional
+	Spec JaegerSpec `json:"spec,omitempty"`
+
+	// +optional
 	Status JaegerStatus `json:"status,omitempty"`
 }
 
 // JaegerCommonSpec defines the common elements used in multiple other spec structs
-// +k8s:openapi-gen=true
+// +k8s:openapi-gen=false
```

Comment:
Why is this now false?

Reply:
I'll try reverting this and testing again, but I remember openapi complaining about this type at the early stages of this PR.

Reply:
Setting it to `true` generates no complains from openapi. I reverted this change.